Sources/IO/Geometry/GLTFImporter/index.d.ts

```diff
@@ -34,6 +35,8 @@ export interface vtkGLTFImporter extends vtkGLTFImporterBase {
 
   parse(content: string): void;
 
+	buildActorFromPrimitive(primitive: gltfPrimitive): vtkActor;
```

Comment:
tabs are indeed mandatory in index.d.ts files. Please eventually fix the other indentations to tabs (can be done in a separate PR)